terraform/main.tf

```diff
@@ -184,7 +184,7 @@ module "aws_deploy-ap-southeast-1" {
   spot_nodes   = 14
 
   spot_price    = "0.125"
-  instance_type = "m4.large"
+  instance_type = "t3.small"
```

Comment:
We can't mine on this instance, we need 4GB+ memory and more stable CPU ?

Reply:
ok did not know that we have diffrent node types :D so do we need to run some test or just change it to something cheaper with 4gb of mem?